main.tf

```diff
@@ -32,12 +27,12 @@ resource "null_resource" "ansible_playbook" {
       "-i '${local.ip_addresses},'",
       "--private-key=${var.ssh_key_path}",
       "--user=${var.ssh_username}",
-      "--ssh-common-args='-o StrictHostKeyChecking=no ${local.ansbile_ssh_proxy}'",
+      "--ssh-common-args='-o StrictHostKeyChecking=no ${local.ansible_ssh_proxy}'",
       "-e 'cert_names=${local.cert_names}'",
-      "-e 'local_output_dir=${local.working_dir}'",
+      "-e 'local_output_dir=${var.working_dir}/ansible.${self.id}'",
```

Comment:
No longer need `local_file.current_work_directory` because it's implicitly created? Can ansible manage it?

Reply:
Yeah - I think it's cleaner that way